cmd/telemetrygen/internal/logs/logs.go

```diff
@@ -90,6 +105,32 @@ func Run(c *Config, exp exporter, logger *zap.Logger) error {
 	return nil
 }
 
+// Validate validates the test scenario parameters.
+func Validate(c *Config) error {
```

Comment:
I think we will change to use confmap eventually, so better to follow that pattern:
```suggestion
func (c *Config) Validate error {
```

Reply:
good idea, I changed the code to reflect that